shared/OpenTelemetry/src/enums/logs/OTelSeverityNumber.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Numerical values of severity levels.
+ */
+export const enum OTelSeverityNumber {
```

Comment:
If we want to have an "exported" OTelSeverityNumber then this still needs some work.

The constant enum shoudl start with a lowercase "e" and then a type with this name etc 
see https://github.com/microsoft/ApplicationInsights-JS/blob/otel-sdk/shared/OpenTelemetry/src/enums/eAttributeChangeOp.ts for an example. This produces an exported "type" which matches what TypeScript produces by having a non-const enum but with far less generated code.